include/SDL3/SDL_gpu.h

```diff
@@ -1514,7 +1514,43 @@ extern DECLSPEC void SDLCALL SDL_GpuEndComputePass(
 	SDL_GpuComputePass *computePass
 );
 
-/* TransferBuffer Set/Get */
+/* TransferBuffer Data */
+
+/**
+ * Maps a transfer buffer into application address space.
+ * You must unmap the transfer buffer before encoding upload commands
+ * or calling SetTransferData or GetTransferData.
+ *
+ * \param device a GPU context
+ * \param transferBuffer a transfer buffer
+ * \param offsetInBytes offset from the beginning of the transfer buffer
+ * \param sizeInBytes number of bytes to map
+ * \param cycle if SDL_TRUE, cycles the transfer buffer if it is bound
+ * \param ppData a pointer to an application-accessible pointer
```

Comment:
would rephrase this like
`where to store the address of the mapped transfer buffer memory`